CHANGES/883.bugfix

```diff
@@ -0,0 +1,3 @@
+Started treating string ``port`` passed into :py:meth:`~yarl.URL.build` as TypeError -- :user:`commonism`.
+
+The logic has changed to create URL with no colon and port rendered in its string representation.
```

Comment:
I suppose this isn't really true anymore?

Reply:
Looks like you marked this as resolved by accident, without actually updating anything...

Reply:
TypeError was an update.